src/character.h

```diff
@@ -484,6 +484,7 @@ class Character : public Creature, public visitable<Character>
         std::vector<bionic> my_bionics;
 
     protected:
+        virtual void on_stat_change( const std::string &, int ) override {};
```

Comment:
Pretty sure `mod_stat` looks like this to make lua interface easier to use. But yeah, it shouldn't really be used for non-lua part of the code.